src/app/frontend/deploy/helpsection/helpsection.scss

```diff
@@ -17,6 +17,7 @@
 // This rule is meant to align items only in the current layout (kd-help-section).
 .kd-help-section > * {
   flex: 1;
+  font-size: 14px;
```

Comment:
Instead of doing this, just apply md-body-1 class to the entire deploy form page. E.g., to its root element.